test/units/cli/test_adhoc.py

```diff
@@ -93,7 +93,34 @@ def test_run_no_extra_vars():
     assert exec_info.value.code == 2
 
 
-def test_ansible_version(capsys, mocker):
+def test_ansible_git_version(capsys, mocker):
+    mocker.patch("ansible.cli.arguments.option_helpers._git_repo_info",
+                 return_value="ansible 2.11.0.dev0 (devel 8a202cae3e) last updated 2021/01/11 10:24:38 (GMT +200)")
```

Comment:
This test is basically a copy-paste of the old one. It does exactly the same thing. Instead of making a copy, use `@pytest.mark.parametrize`.